sphinx/builders/__init__.py

```diff
@@ -79,8 +79,7 @@ def __init__(self, app):
         self.confdir = app.confdir
         self.outdir = app.outdir
         self.doctreedir = app.doctreedir
-        if not path.isdir(self.doctreedir):
-            os.makedirs(self.doctreedir)
```

Comment:
This change causes flake8 error.
```
./sphinx/builders/__init__.py:12:1: F401 'os' imported but unused
```
https://travis-ci.org/sphinx-doc/sphinx/jobs/313209844#L663

Could you fix it please?

Reply:
I'm sorry about that. Pushed a fixed version now.